### PR TITLE
Improve inter-guide reading experience; Staking Cleanup

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: docs/source/guides/staking_guide.rst [hunk 1]
```diff
@@ -4,16 +4,13 @@
 NuCypher Staking Guide
 =======================
 
-The NuCypher Network status page can be found at https://status.nucypher.network:12500/.
+Participation in staking nucypher is two divided into two primary roles: "Staker" and "Worker":
+The account which holds NU tokens and manages stakes is called a "Staker", and the account
+which participates in the network as an Ursula node is called "Worker". The recommended configuration that
```
Comment (mswilkison) [inline docs/source/guides/staking_guide.rst]:
```suggestion
which participates in the network as an Ursula node is called "Worker". The recommended configuration is that
```

File: docs/source/guides/staking_guide.rst [hunk 2]
```diff
@@ -4,16 +4,13 @@
...
+StakeHolder and Worker have separate Ethereum accounts: Staker controlled by a hardware wallet, and Worker by software wallet.
```
Comment (mswilkison) [inline docs/source/guides/staking_guide.rst]:
```suggestion
StakeHolder and Worker have separate Ethereum accounts: Staker controlled by a hardware wallet, and Worker by a software wallet.
```

File: docs/source/guides/staking_guide.rst [hunk 3]
```diff
@@ -4,16 +4,13 @@
...
-The account which is actively doing work for the network (i.e., re-encryptions) needs to be a hot
-wallet. However, it doesn't have to be the same account as the one which receives and
-sends NU tokens. The account which holds NU tokens is called
-Staker, and the account which participates in the network as a worker node is
-called `Ursula`, or simply, Worker.
-It is highly recommended that Staker and Worker have separate Ethereum
-accounts: Staker controlled by a hardware wallet, and Worker
-having an address controlled by geth.
+Note: While it is possible to use the same account, It is highly recommended that the
+Staker and Worker have separate Ethereum accounts: Staker controlled by a hardware wallet, and Worker by software.
```
Comment (mswilkison) [inline docs/source/guides/staking_guide.rst]:
```suggestion
Staker and Worker have separate Ethereum accounts: Staker controlled by a hardware wallet, and Worker by a software wallet.
```

File: docs/source/guides/staking_guide.rst [hunk 4]
```diff
@@ -302,6 +300,54 @@ allow **re-staking** to be disabled until the release period begins, even if you
...
+
+.. code:: bash
+
+    (nucypher)$ nucypher stake prolong
+
+
+Wind Down
+**********
+
+To set the winding down time of an existing stake:
```
Comment (mswilkison) [inline docs/source/guides/staking_guide.rst]: I'm not sure this is a correct description. The winddown time is equal to the stake duration. Running `nucypher stake winddown` starts the winddown but it doesn't set the time. At least, that's my understanding
```suggestion
To start winding down an existing stake:
```

File: docs/source/guides/staking_guide.rst (hunk 3 above)
Comment (cygnusv) [inline docs/source/guides/staking_guide.rst]: This is the same text than the end of last paragraph.

File: docs/source/guides/staking_guide.rst [hunk 5]
```diff
@@ -302,6 +300,54 @@ allow **re-staking** to be disabled until the release period begins, even if you
...
+|  ``prolong``         | Prolong an existing stake's duration                                          |
++----------------------+-------------------------------------------------------------------------------+
+|  ``winddown``        | Manage winding down of stakes                                                 |
++----------------------+-------------------------------------------------------------------------------+
+
+
+Divide
+******
+
+To divide an existing stake into discrete substakes:
```
Comment (cygnusv) [inline docs/source/guides/staking_guide.rst]: Link to substakes docs.

File: docs/source/guides/ursula_configuration_guide.rst [hunk 6]
```diff
@@ -46,59 +40,32 @@ You need to create a software-controlled account in geth:
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code:: bash
-
-    $ sudo apt install build-essential python3-dev python3-pip
+A fully synced ethereum node or "provider" is required for the worker to read and write to nucypher's smart contracts.
 
-Install ``nucypher`` either by doing ``sudo pip3 install nucypher`` if you have
-a dedicated instance or container, or with a ``virtualenv``:
+In order to be a successful Ursula operator, You will need a machine (physical or virtual) which
```
Comment (cygnusv) [inline docs/source/guides/ursula_configuration_guide.rst]:
```suggestion
In order to be a successful Ursula operator, you will need a machine (physical or virtual) which
```

File: docs/source/guides/ursula_configuration_guide.rst [hunk 7]
```diff
@@ -46,59 +40,32 @@ You need to create a software-controlled account in geth:
...
+can be kept online consistently without interruption and externally accessible via TCP port 9151.
+The well-behaved worker will accepts work orders for re-encryption at-will, and be rewarded as a result.
```
Comment (cygnusv) [inline docs/source/guides/ursula_configuration_guide.rst]:
```suggestion
The well-behaved worker will accept work orders for re-encryption at-will, and be rewarded as a result.
```

File: docs/source/guides/ursula_configuration_guide.rst [hunk 8]
```diff
@@ -176,33 +122,41 @@ Verify that the node setup was successful by running the ``status`` command.
...
+Assuming geth is running locally on goerli, configure and run an Ursula using port and volume bindings:
 
 .. code:: bash
 
-    (nucypher)$ nucypher ursula run
+    export NUCYPHER_KEYRING_PASSWORD=<your keyring password>
+    export NUCYPHER_WORKER_ETH_PASSWORD=<your eth account password>
+
+    # Interactive Ursula-Worker Initialization
+    docker run -it -v ~/.ethereum:/root/.ethereum -v ~/.local/share/nucypher:/root/.local/share/nucypher -e NUCYPHER_KEYRING_PASSWORD nucypher:latest nucypher ursula init --provider file:///root/.ethereum/goerli/geth.ipc --staker-address <YOUR STAKING ADDRESS>
```
Comment (cygnusv) [inline docs/source/guides/ursula_configuration_guide.rst]:
```suggestion
    docker run -it -v ~/.ethereum:/root/.ethereum -v ~/.local/share/nucypher:/root/.local/share/nucypher -e NUCYPHER_KEYRING_PASSWORD nucypher:latest nucypher ursula init --provider file:///root/.ethereum/goerli/geth.ipc --staker-address <YOUR STAKING ADDRESS> --network <NETWORK_NAME>
```

File: docs/source/guides/installation_guide.md [hunk 9]
```diff
@@ -14,11 +14,16 @@
...
 * We have tested `nucypher` with Windows, Mac OS, and GNU/Linux (GNU/Linux is recommended).
 * If you don’t already have it, install [Python](https://www.python.org/downloads/). As of November 2019, we are working with Python 3.6, 3.7, and 3.8.
-* We also require the following system packages (Linux):
 
-    - `libffi-dev`
-    - `python3-dev`
-    - `python3-virtualenv`
+Before installing ``nucypher``, you may need to install necessary developer
+tools and headers, if you don't have them already. In Ubuntu, Debian, Linux Mint
+or similar distros, that is:
```
Comment (michwill) [inline docs/source/guides/installation_guide.md]: To make it simple, we can recommend `sudo apt install python3-dev build-essential libffi-dev python3-pip` as one command

File: docs/source/guides/ursula_configuration_guide.rst [hunk 10]
```diff
@@ -176,33 +122,41 @@ Verify that the node setup was successful by running the ``status`` command.
...
+
+    # Daemonized Ursula
+    docker run -d -v ~/.ethereum:/root/.ethereum -v ~/.local/share/nucypher:/root/.local/share/nucypher -p 9151:9151 -e NUCYPHER_KEYRING_PASSWORD -e NUCYPHER_WORKER_ETH_PASSWORD nucypher/nucypher:latest nucypher ursula run --teacher discover.nucypher.network:9151 --provider file:///root/.ethereum/goerli/geth.ipc
```
Comment (cygnusv) [inline docs/source/guides/ursula_configuration_guide.rst]:
```suggestion
    docker run -d -v ~/.ethereum:/root/.ethereum -v ~/.local/share/nucypher:/root/.local/share/nucypher -p 9151:9151 -e NUCYPHER_KEYRING_PASSWORD -e NUCYPHER_WORKER_ETH_PASSWORD nucypher/nucypher:latest nucypher ursula run 
```

File: docs/source/guides/staking_guide.rst [hunk 11]
```diff
@@ -4,16 +4,13 @@
 NuCypher Staking Guide
 =======================
 
-The NuCypher Network status page can be found at https://status.nucypher.network:12500/.
+Participation in staking nucypher is divided into two primary roles: "Staker" and "Worker":
+The account which holds NU tokens and manages stakes is called a "Staker", and the account
+which participates in the network as an Ursula node is called "Worker". The recommended configuration is that
+StakeHolder and Worker have separate Ethereum accounts: Staker controlled by a hardware wallet, and Worker by a software wallet.
```
Comment (derekpierre) [inline docs/source/guides/staking_guide.rst]:
```suggestion
the Staker and Worker have separate Ethereum accounts: Staker controlled by a hardware wallet, and Worker by a software wallet.
```

File: docs/source/guides/installation_guide.md [hunk 12]
```diff
@@ -64,12 +73,17 @@ Here is the recommended procedure for setting up `nucypher` in this fashion:
...
 
-    In the console:
+    Before continuing, verify that your ``nucypher`` installation and entry points are functional;
+    Activate your virtual environment (if you haven't already) and run the ``nucypher --help`` command in the console:
 
     ```bash
     nucypher --help
     ```
 
+    You will see a list of possible usage options (``--version``, ``-v``, ``--dev``, etc.) and commands (``status``, ``ursula``).
```
Comment (derekpierre) [inline docs/source/guides/installation_guide.md]:
```suggestion
    You will see a list of possible usage options (``--version``, ``-v``, ``--dev``, etc.) and commands (``status``, ``ursula``, etc.).
```

File: docs/source/guides/ursula_configuration_guide.rst [hunk 13]
```diff
@@ -4,40 +4,34 @@
...
-
-Run a local geth node on goerli using volume bindings:
+Run a local geth node on Görli using volume bindings:
 
 .. code:: bash
 
     docker run -it -p 30303:30303 -v ~/.ethereum:/root/.ethereum ethereum/client-go --goerli
 
-For alternate methods of running geth via docker see: `Geth Docker Documentation <https://geth.ethereum.org/docs/install-and-build/installing-geth#run-inside-docker-container>`_.
+For alternate geth configuration geth via docker see:
```
Comment (derekpierre) [inline docs/source/guides/ursula_configuration_guide.rst]:
```suggestion
For alternate geth configuration via docker see:
```

File: docs/source/guides/ursula_configuration_guide.rst [hunk 14]
```diff
@@ -46,59 +40,32 @@ You need to create a software-controlled account in geth:
...
 
-.. code:: bash
-
-    $ sudo apt install build-essential python3-dev python3-pip
+A fully synced ethereum node or "provider" is required for the worker to read and write to nucypher's smart contracts.
 
-Install ``nucypher`` either by doing ``sudo pip3 install nucypher`` if you have
-a dedicated instance or container, or with a ``virtualenv``:
+In order to be a successful Ursula operator, you will need a machine (physical or virtual) which
+can be kept online consistently without interruption and externally accessible via TCP port 9151.
```
Comment (derekpierre) [inline docs/source/guides/ursula_configuration_guide.rst]:
```suggestion
can be kept online consistently without interruption and is externally accessible via TCP port 9151.
```

File: docs/source/guides/staking_guide.rst (hunk 5 above)
Comment (KPrasch) [reply docs/source/guides/staking_guide.rst]: :heavy_check_mark: (above)

**Pull request merged**